### DISCUSSION TITLE
Mantling (ledge climbing)

### DISCUSSION THREAD

**Discussion by poperigby** [redeclipse/base#1174] [Help and Support]
I think it could be a good idea to have mantling over objects, like in [Doom 2016](https://www.youtube.com/watch?v=9aJHO2aKmks).  There are lots of times where it would make sense, like when you're trying to get over a low object, or you want to fling yourself over the edge at the last second during a vertical wall run. An accompanying animation where your arms grab onto the ledge would also be cool.

**Comment by qreeves** [comment]
Press `Q`.

**Comment by poperigby** [comment]
Oh. It should probably have an animation. Like with sliding, it's  a bit difficult to tell when you're doing it because it has no special animation. It looks like every other move you do.

**Comment by qreeves** [comment]
Can't do much about it unless an artist donates their time to do that.
